datafusion/expr/src/function_err.rs

```diff
@@ -1,125 +0,0 @@
-// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This is recently added code from @2010YOUY01 (which was awesome) which I simply moved into `BuiltInScalarFunction`